src/Raven.Client/Documents/Queries/QueryResultBase.cs

```diff
@@ -58,6 +58,8 @@ public abstract class QueryResultBase<TResult, TInclude>
         /// <value><c>true</c> if the index is stale; otherwise, <c>false</c>.</value>
         public bool IsStale { get; set; }
 
+        public bool IsMapReduce { get; set; }
```

Comment:
in order to know if it's a map-reduce index so we can reduce the results in the orchstrator.
this can be removed because we can know this in the orchestrator.